src/lib.rs

```diff
@@ -1,203 +1,260 @@
+//! # clap
```

Comment:
It seems like this is copy of `README.md` file. There could be a way to incorporate the `README.md` file in here, so that it does not need be manually updated.